src/qinfer/resamplers.py

```diff
@@ -61,7 +62,7 @@
 
 class Resampler(with_metaclass(ABCMeta, object)):
     @abstractmethod
-    def __call__(self,  model, particle_weights, particle_locations,
+    def __call__(self,  model, particle_dist,
```

Comment:
Right, given that I don't think anyone but us have really used custom resampling yet, I think it's quite safe to make the breaking change that @dseuss suggests here.